apps/workflow-cf.yml.j2

```diff
@@ -55,11 +57,15 @@ Resources:
           {% endif %}
         JobRoleArn: !Ref TaskRoleArn
         ExecutionRoleArn: !GetAtt ExecutionRole.Arn
+        NetworkConfiguration:
+          AssignPublicIp: ENABLED
```

Comment:
What's this for?

Reply:
Fargate jobs running in a public subnet require a public ip address for outbound internet access, see https://docs.aws.amazon.com/AmazonECS/latest/userguide/fargate-task-networking.html

I'm not sure if fargate jobs running in a private subnet with a NAT (e.g. Earthdata Cloud) require a public ip, but I did an ad-hoc deployment into our earthdata cloud uat account and jobs ran fine there with this setting enabled.